lib/vdyp-common/src/main/java/ca/bc/gov/nrs/vdyp/io/parse/control/ControlFileParser.java

```diff
@@ -96,7 +97,7 @@ public Map<String, Object> parse(InputStream input, Map<String, Object> control)
 		result = lineParser.parse(input, result, (map, r, line) -> {
 			Integer index = (Integer) map.get("index");
 			String extend = (String) map.get("extend");
-			String restOfLine = (String) map.get("restOfLine");
+			String restOfLine = Utils.<String>optSafe(map.get("restOfLine")).orElse("");
```

Comment:
I considered doing the fix in LineParser to make sure this would never be null, but other code was filtering based on it so I made it null safe here instead.